docs/en/query_language/agg_functions/reference.md

```diff
@@ -1,12 +1,55 @@
-
 # Function reference
 
-## count() {#agg_function-count}
+## count {#agg_function-count}
+
+Counts the number of rows.
+
+```
+count()
```

Comment:
In SQL standard and most other databases this is written as `COUNT(*)`, which is also supported by ClickHouse and worth mentioning (0 arguments is ClickHouse-specific extension).

Also `count` can get arbitrary expression as first argument, which means "count how many times this expression returned not null". So for non-nullable expressions it's the result will be the same as for `count()`, but for nullable it differs. The result for nullable argument will still be non-nullable (0 if all rows were null), which differs from other aggregate functions where nullable argument usually makes result nullable as well.